psutil/arch/solaris/process_as_utils.c

```diff
@@ -0,0 +1,302 @@
+/*
```

Comment:
But why? All functions here are works only with address space and likely useless anywhere else. If some other code which works with process address space will appear it can land here.